SharePointDsc/DSCResources/MSFT_SPSite/MSFT_SPSite.psm1

```
@@ -182,6 +182,19 @@ function Get-TargetResource
+            # The RootWeb Name property is empty on an SPWeb object; the site collection name is
+            # held in the Title property. Use the multilingual TitleResource for the web's UI
+            # culture when available, falling back to the plain Title.
+            $rootWebName = $site.RootWeb.Title
+            try
+            {
+                $rootWebName = $site.RootWeb.TitleResource.GetValueForUICulture($site.RootWeb.UICulture)
+            }
+            catch
+            {
+                # Fall back to the plain Title when TitleResource / UICulture is unavailable.
+            }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)